packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -319,9 +319,22 @@ export interface IGetAvailableSlots {
       emoji?: string | undefined;
     }[]
   >;
+  teamMember?: string | undefined;
 }
 
 export async function getAvailableSlots({ input, ctx }: GetScheduleOptions): Promise<IGetAvailableSlots> {
+  const bookerCalUser = input.bookerEmail
```

Comment:
In case of collective events, or if you have added additional guests, can you check their availabilities as well?